ra/registration-authority_test.go

```diff
@@ -204,7 +225,48 @@ func initAuthorities(t *testing.T) (core.CertificateAuthority, *DummyValidationA
 	AuthzFinal.Expires = &exp
 	AuthzFinal.Challenges[0].Status = "valid"
 
-	return &ca, va, sa, &ra
+	return &ca, va, ssa, &ra, cleanUp
+}
+
+// This is an unfortunate bit of tech debt that is being taken on in
```

Comment:
Added a TODO that referenced the newly made https://github.com/letsencrypt/boulder/issues/629